yarn.lock

```diff
@@ -2,6 +2,30 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.1.0":
+  version "2.2.0"
+  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.2.0.tgz#56c133824780de3174aed5ab6834f3026790154d"
+  integrity sha512-qRmjj8nj9qmLTQXXmaR1cck3UXSRMPrbsLJAasZpF+t3riI71BXed5ebIOYwQntykeZuhjsdweEc9BxH5Jc26w==
+  dependencies:
+    "@jridgewell/gen-mapping" "^0.1.0"
+    "@jridgewell/trace-mapping" "^0.3.9"
+
+"@babel/cli@^7.5.5":
+  version "7.18.6"
```

Comment:
We're still seeing 7.18.6, since that's valid with the semantic version we've chosen. Should we upgrade the other dependencies, too?

Reply:
Oh interesting, is it supposed to match up?? And for the other dependencies, are you referring to the `dependencies`/`optionalDependencies` listed under this `@babel/cli`?

Reply:
Ideally, the versions of all the `@babel/*` dependencies should match up. Otherwise, the CLI may be using a different version/binary than the main build.

Reply:
Ohhhhh I see....let me see...

Reply:
@jstoffan Huh..it seems that installing the `@babel/cli@^7.5.5` version results in this version mismatch...but installing `@babel/cli@7.5.5` matches everything up.